schemaTypes/ProductType.js

```diff
@@ -0,0 +1,11 @@
+const { GraphQLObjectType, GraphQLNonNull, GraphQLID } = require('graphql');
+
+const ProductType = new GraphQLObjectType({
+  name: 'Product',
```

Comment:
Is this a placeholder? If names are here to stay, it might be worthwhile to decide on the naming convention ('Product' vs 'product') and keep it consistent across schema types